PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.springcloud/src/main/java/com/microsoft/azure/toolkit/eclipse/springcloud/streaminglog/SpringCloudLogStreamingHandler.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.eclipse.springcloud.streaminglog;
+
+import org.eclipse.swt.widgets.Display;
+
+import com.microsoft.azure.toolkit.eclipse.common.logstream.EclipseAzureLogStreamingManager;
+import com.microsoft.azure.toolkit.lib.Azure;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azure.toolkit.lib.springcloud.AzureSpringCloud;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudApp;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudDeployment;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudDeploymentEntity;
+import com.microsoft.azure.toolkit.lib.springcloud.SpringCloudDeploymentInstanceEntity;
+
+public class SpringCloudLogStreamingHandler {
+    public static void startLogStreaming(final SpringCloudApp app) {
+        AzureTaskManager.getInstance().runLater(() -> {
+            final SpringCloudStreamingLogDialog dialog = new SpringCloudStreamingLogDialog(
+                    Display.getCurrent().getActiveShell(), app);
```

Comment:
you can pass deployment directly.

Reply:
Fixed in https://github.com/microsoft/azure-tools-for-java/pull/6070/commits/543bfb626541ad2a08399b21da2b379981a627a8